### ISSUE TITLE
Combine pages into index.html

### ISSUE THREAD

**Issue by clayperez** [Monkeychip/html5App#1]
You might be planning on doing this, but a suggestion:

Move the three main pages "bugs.html", "handjammer.html" both into "index.html" so that everything is in one place and you're only showing/hiding them. One issue I had was that the header links were different and from the "bugs.html" page you couldn't click back to "index.html" -- it's minor, but this could creep.

Plus, each time you click to another HTML page it loads all of the content, all of the scripts, etc -- it's redundant, and can lead to bugs or inconsistencies between pages if you don't update libraries or links in all three places each time you make a change to some shared component.

**Comment by Monkeychip** [comment]
Good idea.  I noticed it was a bit buggy and that makes a lot of sense.
